src/components/comment/CommentForm.jsx

```diff
@@ -30,24 +30,25 @@ const CommentForm = (props) => {
     }
 
     return (
-        <Form onSubmit={submitHandler} className="comment-form" onKeyUp={onKeyUpHandler} onClick={onClickHandler}>
-            <Form.Group className="mb-3" controlId="formBasicComment">
-                <Col className="col-lg-12">
-                    <Row className="col-lg-12">
-                        <Form.Control
-                            name="comment"
-                            as="textarea"
-                            placeholder="Write your comments here  (Ctrl+Enter to confirm)"
-                            required
-                            value={commentValue}
-                            onChange={onValueChange}
-                            ref={formInputRef}
-                        />
-                    </Row>
-                    <Row className="send-comment-arrow col-lg-12">
-                        <Button className="comment-button" variant="primary" type="submit" >
-                            <ArrowRight />
-                        </Button>
+        <Form onSubmit={submitHandler} onKeyUp={onKeyUpHandler} onClick={onClickHandler}>
```

Comment:
I suggest to keep convention and have onSubmitHandler:
onKeyUp={onKeyUpHandler} 
onClick={onClickHandler}
onSubmit={**onS**ubmitHandler}